index.html

```diff
@@ -1,80 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rock, Paper, Scissors</title>
-</head>
+  </head>
 
-<body>
+  <body>
 
-<script>
-  
-function computerPlay() {
-    let rand = Math.random();
-    if (rand < 0.34) {
+    <script>
+    
+    function computerPlay() {
+      let rand = Math.random();
+      if (rand < 0.34) {
         return 'rock';
-    }else if(rand <= 0.67) {
+      } else if (rand <= 0.67) {
         return 'paper';
-    }else{
+      } else {
         return 'scissors';
+      }
     }
-}
 
-let playerScore = parseInt(0);
-let computerScore = parseInt(0);
+    let playerScore = parseInt(0);
+    let computerScore = parseInt(0);
 
-function playRound(playerSelection, computerSelection) {
-if(playerSelection.toLowerCase() === 'rock'){
-    if(computerSelection === 'paper'){
-        computerScore++;       
-    }else if(computerSelection === 'scissors'){
-        playerScore++;       
-    }else{
-        return;
-    }
-}
+    function playRound(playerSelection, computerSelection) {
+      if (playerSelection.toLowerCase() === 'rock') {
+        if (computerSelection === 'paper') {
+          computerScore++;       
+        } else if (computerSelection === 'scissors') {
+          playerScore++;       
+        } else {
+          return;
+        }
+      }
 
-if(playerSelection.toLowerCase() === 'paper'){
-    if(computerSelection === 'paper'){
-        return;
-    }else if(computerSelection === 'scissors'){
-        computerScore++;
-    }else{
-        playerScore++;
-    }
-}
+      if (playerSelection.toLowerCase() === 'paper') {
+        if (computerSelection === 'paper') {
+          return;
+        } else if (computerSelection === 'scissors') {
+          computerScore++;
+        } else {
+          playerScore++;
+        }
+      }
 
-if(playerSelection.toLowerCase() === 'scissors'){
-    if(computerSelection === 'paper'){
-        playerScore++;
-    }else if(computerSelection === 'scissors'){
-        return;
-    }else{
-        computerScore++;
+      if (playerSelection.toLowerCase() === 'scissors') {
```

Comment:
Can you think of a way of refactoring these conditionals so that you don't end up with nested if statements?